nix/sources.nix

```diff
@@ -10,29 +10,50 @@ let
     let
```

Comment:
Niv suggested to upgrade the file as well, so the diff here is just the result of `niv init`.